vcpkg/get_started/get-started-vscode.md

```diff
@@ -61,7 +67,7 @@ $env:PATH = "$env:VCPKG_ROOT;$env:PATH"
 ::: zone pivot="shell-cmd"
 
 ```console
-set VCPKG_ROOT="C:\path\to\vcpkg"
+set "VCPKG_ROOT=C:\path\to\vcpkg"
```

Comment:
This fix needs to be replicated in other places since otherwise the VCPKG_ROOT variable will contain the quotation marks.